src/main/java/com/teamscale/upload/report/xcode/ActionRecord.java

```diff
@@ -0,0 +1,23 @@
+package com.teamscale.upload.report.xcode;
+
+/**
+ * An {@link ActionRecord} in an {@link ActionsInvocationRecord}.
```

Comment:
Since there is no documentation on the output format I just derived it from the type annotations in the JSON output. Not sure why all this nesting with multiple objects exists but we have to deal with it somehow.